.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: Test collection
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  ansible-lint:
+    uses: ansible-network/github_actions/.github/workflows/ansible-lint.yml@main
+  changelog:
+    uses: ansible-network/github_actions/.github/workflows/changelog.yml@main
+  sanity:
+    uses: ansible-network/github_actions/.github/workflows/sanity.yml@main
+  unit-galaxy:
+    uses: ansible-network/github_actions/.github/workflows/unit_galaxy.yml@main
+  unit-source:
+    uses: ansible-network/github_actions/.github/workflows/unit_source.yml@main
```

Comment:
It's unclear what the difference between "unit-galaxy" and "unit-source" is

Reply:
unit-galaxy : The collections (on which the collection being tested depends) are installed from the latest version on galaxy
unit-source: The collections (on which the collection being tested depends) are installed from the head of the repo.

For amazon.aws,  both these workflows are not different as it does not have any collections dependency (correct me if I am wrong). I can remove one of them.